howto_conda_package.md

```diff
@@ -0,0 +1,361 @@
+The folder option specifies the destination path. `conda-build` will create this within the source-path and save the source information here. Default is the `<packagename>-<version>`. 
+
+#### **The Build Section**
+```
+build:
+    number: 0  
+    noarch: python     
+    script: {{ PYTHON }} hypro-{{ version }}/setup.py install --single-version-externally-managed --record=record.txt
+```
+
+Define all build information in here. The build number should be incremented for every build command. If using 'script', it replaces your build.sh/bld.bat. You should remove those files in this case. Noarch should be used for pure python packages.
+Many more options like defining RPATHs or entry points of python scripts can be found on the mentioned website.
+
+#### **The Requirements Section**
+
+```
+  build:
+    - git
+    - pip==19.3 
+    - python==3.7.6
+    - setuptools
+```
+
+Specify all requirements that are necessary for building the package. This means tools required loading the source code (git) and installing the tool binaries (python, setuptools). You may specify the exact version of the programs as min/max required (>/<=) or an specific version (==).
+
+```
+  host:         
+    - python==3.7.6
+    - pip==19.3 
+    - prokka>=1.14.6
+```
+
+```
+  run:
+    - python==3.7.6
+    - mygene==3.1.0
+    - pandas==0.25.2
+    - mmseqs2==10.6d92c
+    - prokka>=1.14.6
+```
+
+Define the tools for executing your program. Conda will load them in advance, as happened with the build requirements. E.g. mmseqs2 and prokka. In case of python, this includes all non-standard libraries (standard are e.g. sys, os, argparse). In my case, the python script utilizes pandas and the mygene python API. The libraries can then be found by setuptools when executing your setup.py script.
+Note, that the host section was skipped for the HyPro package - it was not required.
+
+#### **The About Section**
+
+```
+about:
+  home: https://github.com/hoelzer-lab/hypro.git
+  summary: Extend hypothetical prokka protein annotations using additional homology searches against larger databases
+  license: GPL
+```
+
+Here you add specifying information about the package. Where is the package maintained and under which license it was published, in this case.
+
+Note: If you want to make your package available on different systems, you should make use of selectors. `conda-build` notices on which system it is run, so the selectors help to decide which information to use and which not. A very convinient option you should consider in case!

```

Comment:
"A very convinient option you should consider in case!"

sounds like Yoda language :D

ah and it's 'convenient'!